rotkehlchen/data_migrations/migrations/migration_3.py

```diff
@@ -1,3 +1,4 @@
+import imghdr
```

Comment:
Don't use imghdr. It's deprecated and will be removed from the standard library.

https://peps.python.org/pep-0594/#imghdr
https://docs.python.org/3/library/imghdr.html

This is something we can probably achieve on our own without a dependency but if it's a small cross-platform, python only dependency we can also introduce one.


![2022-05-21_12-49](https://user-images.githubusercontent.com/1658405/169648505-d6adacde-6d87-490d-ae2f-e2f8b95f7f61.png)

The PEP that deprecates it mentions 3 alternatives:

https://pypi.org/project/filetype/
https://pypi.org/project/puremagic/
https://pypi.org/project/python-magic/

From the above only filetype and puremagic are cross-platform and dependency free, since python-magic is an interface to libmagic, a unix only tool.

And from the 2 it seems that filetype has the most stars and usage, so probably go with it?